providers/trello.js

```diff
@@ -0,0 +1,733 @@
+        this.payload.addEmbed(embed);
+    }
+
+    // How to Trigger?
+    async createChecklist(){
+        
+    }
+
+    async createList(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Created List';
+        embed.description = '`' + this.action.data.list.name + '` has been created.';
+        embed.url = this._resolveFullBoardURL(this.action.data.board);
+        this.payload.addEmbed(embed);
+    }
+
+    async createOrganization(){
+        let embed = this._preparePayload();
+        embed.title = 'Created Organization';
+        embed.description = '`' + this.action.data.organization.name + '` has been created.';
+        embed.url = this._resolveGenericURL(this.action.data.organization.id);
+        this.payload.addEmbed(embed);
+    }
+
+    // Won't Trigger?
+    async createOrganizationInvitation(){
+        
+    }
+
+    async deleteAttachmentFromCard(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Removed Attachment from "' + this.action.data.card.name + '"';
+        embed.url = this._resolveFullCardURL(this.action.data.card);
+        this._formatAttachment(this.action.data.attachment, embed);
+        this.payload.addEmbed(embed);
+    }
+
+    // Won't Trigger?
+    async deleteBoardInvitation(){
+        
+    }
+
+    async deleteCard(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Deleted Card';
+        embed.description = 'A card was deleted from list `' + this.action.data.list.name + '`.';
+        embed.url = this._resolveFullBoardURL(this.action.data.board);
+        this.payload.addEmbed(embed);
+    }
+
+    async deleteCheckItem(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Deleted Check Item from "' + this.action.data.card.name + '"';
+        embed.url = this._resolveFullCardURL(this.action.data.card);
+        embed.description = '`' + this.action.data.checkItem.name + '` was removed from checklist `' + this.action.data.checklist.name + '`.';
+        this.payload.addEmbed(embed);
+    }
+
+    async deleteLabel(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Deleted Label';
+        embed.url = this._resolveFullBoardURL(this.action.data.board);
+        this.payload.addEmbed(embed);
+    }
+
+    // Won't Trigger?
+    async deleteOrganizationInvitation(){
+        
+    }
+
+    async disablePlugin(){ // TODO
+        let embed = this._preparePayload();
+
+        this.payload.addEmbed(embed);
+    }
+
+    // How to Trigger?
+    async disablePowerUp(){
+        
+    }
+
+    // How to Trigger?
+    async emailCard(){
+        
+    }
+
+    async enablePlugin(){ // TODO
+        let embed = this._preparePayload();
+
+        this.payload.addEmbed(embed);
+    }
```

Comment:
The plugin actions will most likely be complete on the next commit. They are now possible thanks to the async/await format 👍